CHANGELOG.md

```diff
@@ -8,7 +8,17 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 ### Added
+
+## [0.2.3] - 2022-10-05
+### Added
 * Support for MySQL storage backend (#210) (by @MidasLamb)
+* Support for type restrictions in authorization models (#223)
+* Tuple validation against type restrictions in Write API (#232)
+
```

Comment:
Should we add the pgx v5 update?

Reply:
Yep! I'll add it. Missed that one dangit.

Reply:
I think that is an argument for adding stuff to the changelog in each PR, within reason. Not for typo fixes for example. WDYT?